jcl/src/java.base/share/classes/java/lang/StringBuffer.java

```diff
@@ -1716,8 +1716,9 @@ static void initFromSystemProperties(Properties props) {
 	String prop = props.getProperty("java.lang.string.create.unique"); //$NON-NLS-1$
 	TOSTRING_COPY_BUFFER_ENABLED = "true".equals(prop) || "StringBuffer".equals(prop); //$NON-NLS-1$ //$NON-NLS-2$
 
-	String growAggressivelyProperty = props.getProperty("java.lang.stringBufferAndBuilder.growAggressively"); //$NON-NLS-1$
-	growAggressively = "".equals(growAggressivelyProperty) || "true".equalsIgnoreCase(growAggressivelyProperty); //$NON-NLS-1$ //$NON-NLS-2$
+	// growAggressively by default
+	String growAggressivelyProperty = props.getProperty("java.lang.stringBufferAndBuilder.growAggressively", "true"); //$NON-NLS-1$ //$NON-NLS-2$
```

Comment:
Changing the second argument to `""` avoids the call to `Boolean.parseBoolean()` in the normal case where the property is not set.

Reply:
Thanks, updated.